chi2/chinotype.py

```diff
@@ -626,7 +704,7 @@ def chi2_output(self, db):
         if len(self.filter) > 0:
             log.info('Filters: {0}'.format(self.filter))
             if 'ALL' in self.filter: self.filter.remove('ALL')
-            cols, rows = do_log_sql(db, filterStr, self.filter)
+            cols, rows = do_log_sql(db, filterStr)
```

Comment:
`do_log_sql(db, sql, self.filter)` produces an error. If the code ran without error before it's because this functionality was not being exercised. Checked the declaration of `do_log_sql` and saw nothing that would even use a filter arg, the third optional argument seems to be about deciding between `execute()` and `executemany()`. Removed third argument and errors stopped.